.phpstorm.meta.php

```diff
@@ -66,14 +66,17 @@
         'heading_permalink/insert',
         'heading_permalink/max_heading_level',
         'heading_permalink/min_heading_level',
-        'heading_permalink/slug_normalizer',
+        'heading_permalink/slug_length',
```

Comment:
This should probably be renamed to `slug_normalizer/max_length` as I'm not convinced this is actually `heading_permalink` specific configuration:

https://github.com/thephpleague/commonmark/blob/ae78399dfe9eaecdf64a4a0a1b75112ae0f0c9e3/src/Normalizer/SlugNormalizer.php#L40-L43

It should still probably allow a custom length to be passed in `$context`, but fallback to the configuration value when not provided.